app/config.js

```diff
@@ -3,7 +3,7 @@ module.exports = {
   appName: 'NHSD Buying Catalogue Admin',
 
   // Base URL
-  baseUrl: process.env.BASE_URL || '',
+  baseUrl: process.env.BASE_URL || '/admin',
```

Comment:
I think you need to change this back?

Reply:
This is what it should be. When they run it in the cluster they use the defaults. We'll have to override it in our .env file.

But going via /admin should also work locally for you

Reply:
ok you will need to fix the tests then

Reply:
as IF you didn't run them before pushing 😆

Reply:
haha normally I do but I thought ah the tests should be ok for this change. OK I might put it back and then do this later. Think it'll all be related to the relativeUrl stuff we still need to do for Admin